src/mkindex_options.hpp

```diff
@@ -57,12 +55,12 @@ struct LambdaIndexerOptions : public SharedOptions
 // INDEXER
 void parseCommandLine(LambdaIndexerOptions & options, int argc, char const ** argv)
 {
-    std::string programName = "lambda3 " + std::string(argv[0]);
+    std::string programName = "lambda3-" + std::string(argv[0]);
```

Comment:
what is ```lambda3-```? :smile:

Reply:
program names are no longer allowed to contain whitespace in SeqAn3 🤷‍♂